tests/ert3/console/integration/test_cli.py

```diff
@@ -121,3 +122,129 @@ def test_cli_record_load_not_existing_file(tmpdir):
     with patch.object(sys, "argv", args):
         with pytest.raises(SystemExit):
             ert3.console.main()
+
+
+def test_cli_status_no_runs(tmpdir, capsys):
+    workspace = tmpdir / _POLY_WORKSPACE_NAME
+    shutil.copytree(_POLY_WORKSPACE, workspace)
+    workspace.chdir()
+
+    experiments = ert3.workspace.get_experiments(workspace)
+
+    args = ["ert3", "init"]
+    with patch.object(sys, "argv", args):
+        ert3.console.main()
+
+    args = ["ert3", "status"]
+    with patch.object(sys, "argv", args):
+        ert3.console.main()
+
+    captured = capsys.readouterr()
+
+    assert "have run" not in captured.out
+    assert "can be run" in captured.out
+
+    lines = [
+        line.strip()
+        for line in captured.out.splitlines()
+        if "have run" not in line and "can be run" and len(line.strip())
```

Comment:
I'm not entire sure of what the purpose of `... and "can be run" and ...` is 🤔

Reply:
That's a typo that happens to work. Sometimes Python is not such a great language....

Reply:
Actually that whole line is a brain spasm... 😄